src/main/resources/logback.xml

```diff
@@ -60,6 +60,14 @@
         <appender-ref ref="Sentry" />
     </logger>
 
+    <!-- publishing messages can be too verbose logging -->
```

Comment:
I put this in because it was being pretty verbose, making logging slow down and I thought trigger the OOM killer. It may have been a contributing factor but I don't think causal. Still, it was pretty big and pretty useless so I am leaving this in.